CONTRIBUTING.md

```diff
@@ -188,15 +188,19 @@ As subject matter experts, *code owners* approve pull requests on the packages t
 
 Acceptance of contributions follows the consensus-seeking process.
 
-All pull requests must be approved by a *collaborator* before the pull request can be accepted.
+All pull requests must be approved by an assigned *collaborator* before the pull request can be accepted. A *collaborator* will be assigned as the designated reviewer of a pull request within [a reasonable period of time](#response-time) of the pull request creation. Other users may provide reviews, however the *collaborator* assigned to the pull request will have final say on approval.
```

Comment:
This reads to me that we'll have one person who is "uber" responsible for *review*. I think the issue is less around review and ensuring that once things are fully reviewed they are merged in a timely manner. Can we update this to reflect that as a reality? My concern would be that there is confusion and a scenario arises where the person assigned to the PR is expected to merge once _they_ have approved, rather than once all policies are done and all reviewing comments have been resolved in an appropriate manner.